src/sidebar/services/test/view-filter-test.js

```diff
@@ -186,11 +187,18 @@ describe('sidebar/services/view-filter', () => {
     });
   });
 
+  /*
+  Alter conditions to make more sense with cotext
+
+  - allow for short circuit of missing id to prevent the check ojn matches()
+- checking target length makes more sense in the context of code it nests.
+*/
```

Comment:
These comments look like some notes you were writing for yourself that accidentally got left in the PR?